### PR TITLE
Creates content assessor

### LINKED ISSUE #447: Create content assessor
Currently we have an assessor for SEO for running SEO assessments. We need one for content too.

### PR DESCRIPTION
**Pull request by terw-dan**
This creates a content assessor and adds this to the app. 

Fixes #447

### CODE REVIEW

**Comment by atimmer** [comment]
CR Done 👍🏻

File: js/app.js [hunk 1]
```diff
@@ -207,9 +208,16 @@ var App = function( args ) {
 
 	// Set the assessor
 	if ( isUndefined( args.assessor ) ) {
-		this.assessor = new SEOAssessor( this.i18n );
+		this.seoAssessor = new SEOAssessor( this.i18n );
 	} else {
-		this.assessor = args.assessor;
+		this.seoAssessor = args.assessor;
```
Comment (jcomack) [inline js/app.js]: Shouldn't `args.assessor` be `args.seoAssessor`?

File: js/app.js [hunk 2]
```diff
@@ -349,6 +358,33 @@ App.prototype.initSnippetPreview = function() {
...
+	};
+
+	// Pass the assessor result through to the formatter
+	this.seoAssessorPresenter = new AssessorPresenter( {
+		targets: targets,
+		assessor: this.seoAssessor,
+		i18n: this.i18n
+	} );
+
+	targets = {
```
Comment (jcomack) [inline js/app.js]: Feels strange overruling the variable used for the SEO Assessor. Why not create a new `var` or directly pass the targets object to the `AssessorPresenter`?

**Comment by jcomack** [comment]
CR 2 done.

**Comment by jcomack** [comment]
CR 3 done 👍

**Comment by jcomack** [comment]
Acceptance done 👍

**Pull request merged**